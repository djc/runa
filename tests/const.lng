
URL = 'Asara'
YEAR = 1983

def main(name: str, args: array[str]):
	print(URL)
	print(str(YEAR))
