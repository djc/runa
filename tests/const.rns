
URL = 'Asara'
YEAR = 1983

def main():
	print(URL)
	print(str(YEAR))
