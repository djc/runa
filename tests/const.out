http://dirkjan.ochtman.nl/
1983
